pkgs/by-name/ba/bash-language-server/package.nix

```diff
@@ -19,9 +19,8 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-yJ81oGd9aNsWQMLvDSgMVVH1//Mw/SVFYFIPsJTQYzE=";
   };
 
-  pnpmWorkspace = "bash-language-server";
   pnpmDeps = pnpm_8.fetchDeps {
-    inherit (finalAttrs) pname version src pnpmWorkspace;
```

Comment:
An explanation in the commit message for this change would be nice. `pnpmWorkspace` is an attribute added to support here:

https://github.com/NixOS/nixpkgs/pull/323493

Reply:
I remove the commit. I think this should be discussed in your mentioned PR.